# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#goal definition
float32 x_setpoint
float32 y_setpoint
float32 z_setpoint
float32 yaw_setpoint
float32 duration
